apps/desktop-wallet/src/App.tsx

```diff
@@ -219,3 +222,20 @@ const AppContainerStyled = styled.div<{ showDevIndication: boolean }>`
       border: 5px solid ${theme.global.valid};
     `};
 `
+
+const useClearQueryPersistedCacheOnVersionUpdate = () => {
```

Comment:
Is there a reason why this is not extracted out of `App.tsx` like the other hooks?

Reply:
Also, nit rename : `useClearPersistedQueryCacheOnVersionUpdate()`. If we say that it's "persisted", we don't need to say that it's a "cache". WDYT?

Reply:
What do you mean? All hooks used in the App.tsx are in the App.tsx file

Reply:
Rename done and pushed.